server/src/socket/events.ts

```diff
@@ -18,6 +19,7 @@ const socketEventHandler = async (socket: Socket, io: Server) => {
     socket.on(Events.START, () => startGameEvent(io, socket).catch(err => handleError(socket, err)))
     socket.on(Events.PLAY_CARD, (cardId: number) => playCardEvent(io, socket, cardId).catch(err => handleError(socket, err)))
     socket.on(Events.FLIP_CARD, (cardId: number) => flipCardEvent(io, socket, cardId).catch(err => handleError(socket, err)))
+    socket.on(Events.VOTE_CARD, (cardId: number) => voteCardEvent(io, socket, cardId).catch(err => handleError(socket, err)))
```

Comment:
We should refactor how we catch errors from the events, Maybe use a wrapper function.

We could create an array of arrays, something like:

```typescript
const eventHandlers = [
  [Events.PLAY_CARD, (cardId) => playCardEvent(io, socket, cardId)]
  ...
]

eventHandlers.forEach(handler => {
  handler[1].catch(err => handleError(socket, err)
  socket.on(...handler)
]

Reply:
solved in https://github.com/albinr99salt/no-name/pull/69